pkgs/kde/gear/kdenlive/default.nix

```diff
@@ -20,7 +23,7 @@ mkKdeDerivation {
     (
       substituteAll {
         src = ./dependency-paths.patch;
-        inherit mediainfo mlt;
+        inherit mediainfo mlt glaxnimate;
```

Comment:
This does nothing

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/324552